src/targz_sizes.c

```diff
@@ -10,12 +11,39 @@
 
 #include <stdint.h>
 #include <stdio.h>
+#include <stdlib.h>
+#include <getopt.h>
 
 #include <zlib.h>
 
 #define COMPRESSED_BUFFER 102400
 #define TAR_BLOCK_SIZE 512
 
+#ifndef max
+    #define max( a, b ) ( ((a) > (b)) ? (a) : (b) )
```

Comment:
Yes, inline static functions look better.  Addressed in commit: 9a23a4c784eb5fb6828ccdfe9835bd58f67b13da